hack/diagnostics/eck-dump.sh

```diff
@@ -110,6 +110,7 @@ main() {
     get_resources $ns persistentvolumes
     get_resources $ns persistentvolumeclaims
     get_resources $ns services
+    get_resources $ns endpoints
```

Comment:
I wonder if dumping this can become a problem on large installations.

Reply:
You mean because of the high number of items?
It's probably the same cardinality as for Pods.